concourse/tasks/generate-build-id.yaml

```diff
@@ -0,0 +1,16 @@
+---
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: busybox
+
+outputs:
+- name: build-id-dir
+
+run:
+  path: sh
+  args:
+  - -c
+  - "buildid=$(date '+%s'); echo $buildid | tee build-id-dir/build-id; touch build-id-dir/((prefix))-v${buildid}-root.tar.gz"
```

Comment:
i think the prefix-vbuildid-root.tar.gz is wrong now? the -noRoot option instead creates 'prefix-vbuildid.tar.gz' i think. please confirm

Reply:
i confirmed this. you should change to remove "-root" from the touch command